conf/layer.conf

```diff
@@ -10,3 +10,15 @@ BBFILE_PATTERN_meta-nilrt = "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-nilrt = "15"
 
 LAYERSERIES_COMPAT_meta-nilrt = "dunfell"
+
+
+## NILRT GIT REPO CONFIGURATION
+NILRT_GIT_RO_PROTO ?= "git://"
+NILRT_GIT_RW_PROTO ?= "https://"
```

Comment:
Hmm... you're right. I'll drop these configs.

I think it's actually more correct that `NILRT_GIT` be defined in the layer.conf - since it is specific to the layer recipes, rather than the distribution - but that's not important enough to warrant moving it at this time.